src/bun.js/bindings/ImportMetaObject.cpp

```diff
@@ -340,6 +340,8 @@ extern "C" JSC::EncodedJSValue functionImportMeta__resolveSyncPrivate(JSC::JSGlo
                 auto bunStr = Bun::toString(parentIdStr);
                 args.append(jsBoolean(Bun__isBunMain(lexicalGlobalObject, &bunStr)));
 
+                auto thisValue = callFrame.thisValue();
```

Comment:
y tho

Reply:
mistakenly included, it is already removed.